src/hotspot/share/memory/allocation.hpp

```diff
@@ -283,10 +283,10 @@ class CHeapObj {
 
 class StackObj {
  private:
-  void* operator new(size_t size) throw();
-  void* operator new [](size_t size) throw();
-  void  operator delete(void* p);
-  void  operator delete [](void* p);
+  void* operator new(size_t size) throw() = delete;
+  void* operator new [](size_t size) throw() = delete;
```

Comment:
The lingering nothrow exception-specs here are just clutter and can be removed.

Reply:
Done.